Modules/System Tests/Assisted Setup/src/AssistedSetupVisibleTransName.PageExt.al

```diff
@@ -3,7 +3,7 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // ------------------------------------------------------------------------------------------------
 
-pageextension 132588 AssistedSetupVisibleTransName extends "Assisted Setup"
+pageextension 132586 AssistedSetupVisibleTransName extends "Assisted Setup"
```

Comment:
Did this require a id update?

Reply:
With the old ID publishing of System Application Test Libraries fails with the error:
`Your program license does not allow you to publish AssistedSetupVisibleTransName`
You can see it in the logs [here](https://github.com/IhorHandziuk/ALAppExtensions/pull/12).

Since pageextensions piggyback on pages in the license, I changed this ID to be the same as in one of the pages.